### PR TITLE
Simplify schema management further

### PR DESCRIPTION
**Pull request by leo**
This change continues to improve what was landed in https://github.com/ronin-co/compiler/pull/43 by shortening the logic further.

### CODE REVIEW

File: src/utils/model.ts [hunk 1]
```diff
@@ -32,7 +29,7 @@ import {
   prepareStatementValue,
 } from '@/src/utils/statement';
 import title from 'title';
-import type { ModelIndex } from '../../dist';
+import type { ModelIndex, ModelTrigger } from '../../dist';
```
Comment (juriadams) [inline src/utils/model.ts:32]: Should this be imported from `dist/`?

**Review by juriadams — CHANGES_REQUESTED** [review]

File: src/utils/model.ts (hunk 1 above)
Comment (leo) [reply src/utils/model.ts]: That should not be the case! Excellent catch! Seems like VSCode's auto-import made a mistake here (the dist files are git-ignored).

Fixed in [c24844c](https://github.com/ronin-co/compiler/pull/45/commits/c24844ca9799ab82b2bc97156fcbfee43d2b4e50)!

**Comment by leo** [comment]
Applied @juriadams' suggestion and re-requested review!

File: src/utils/model.ts [hunk 2]
```diff
@@ -655,132 +634,80 @@ export const transformMetaQuery = (
...
+      ? (Object.keys(query.alter).filter((key) => key !== 'model')[0] as ModelQueryType)
+      : (queryType as ModelQueryType);
 
-  let queryInstructions: ReturnType<typeof splitQuery>['queryInstructions'] | undefined;
+  const actionReadable =
+    action === 'create' ? 'creating' : action === 'alter' ? 'altering' : 'dropping';
+
+  const entity = (
+    subAltering && query.alter
+      ? Object.keys((query.alter as unknown as Record<ModelQueryType, string>)[action])[0]
```
Comment (leo) [inline src/utils/model.ts:649]: These type castings will be improved in a subsequent iteration.

**Review by juriadams — APPROVED** [review]

**Pull request merged**